### PR TITLE
Track page views and app actions with Google Analytics

### PR DESCRIPTION
**Pull request by jeremiak**
This PR adds functionality to track page views and app actions using Google Analytics. There are two new helper functions in `/static_src/util/analytics.js`:

0. `trackAction()` - takes an `action` object
0. `trackPageView()` - takes a `url` string

The dispatcher (63ed517) is the centralized place for tracking actions though Google Analytics calls them "events", and the router (f5f8a23) tracks page views.

A few open questions for @msecret and @mogul:

0. Are there any actions we should blacklist? That is to say, are there any actions on which we do _not_ track?
0. What is the proper GA property id? [Currently](https://github.com/18F/cg-deck/blob/974d80219a57c5931e125223a7684062cc27c874/static/index.html#L15) I am using the top level cloud.gov ID but we might want to make a new property for the deck. 

Refs #266

### CODE REVIEW

**Comment by mogul** [comment]
> 0. Are there any actions we should blacklist? That is to say, are there any actions on which we do _not_ track?

Not that I know of... Let's try capturing everything first, and see how noisy our data is and decide what to omit later.

> 0. What is the proper GA property id? [Currently](https://github.com/18F/cg-deck/blob/974d80219a57c5931e125223a7684062cc27c874/static/index.html#L15) I am using the top level cloud.gov ID but we might want to make a new property for the deck. 

I believe that's the right one to use... I think we only ever had one for all of cloud.gov, but @msecret may know better.

**Comment by msecret** [comment]
In the original story #266, I said we should ignore actions involved in fetching, and instead ensure we track the whole page view. I don't think it's important that they are ignored, but I do think it would be good to do synthetic page view tracking (because this is a single page app, going to different pages might not trigger a new page view so there's a GA api to track it manually)

**Comment by msecret** [comment]
@jeremiak I was under the impression we should use the one on cg-style, which is the one you have.

File: static_src/util/analytics.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export function trackAction(action) {
+  if (!window.ga) return;
+
+  window.ga('send', {
+    hitType: 'event',
+    eventCategory: action.source,
+    eventAction: action.type,
+    eventLabel: action.type
```
Comment (msecret) [inline static_src/util/analytics.js]: Do we need this if it's repeated? Can't `label` be empty?

Comment (jeremiak) [inline static_src/util/analytics.js]: Yup, totally can. Will remove and update

**Comment by jeremiak** [comment]
@msecret as far as I understand it, f5f8a23d79bfd2ef7135456217b760107a988cc1 should enable synthetic page view tracking by sending a new page view event on every request that matches a route in the router

**Comment by jeremiak** [comment]
This is now ready for code review @msecret

**Pull request merged**